packages/MarkdownEditor-Core.package/MarkdownSettings.class/instance/addStylerMorphsFor..st

```diff
@@ -1,11 +1,12 @@
-tabs
-addStylerMorphsFor: aCategory
+content
+addStylerMorphsFor: aCategory 
 
 	| blockStylerMorphsToAdd |
-	blockStylerMorphsToAdd := self blockStylerMorphs select: [:stylerMorph | 
-		stylerMorph classCategory = aCategory].
-
-	blockStylerMorphsToAdd doWithIndex: [:blockStylerMorph :index |
-		| positionedMorph |
-		positionedMorph := blockStylerMorph position: 20@(index * blockStylerMorph height).
-		self contentMorph scroller addMorphBack: positionedMorph].
\ No newline at end of file
+	
+	blockStylerMorphsToAdd := self blockStylerMorphs
+				select: [:stylerMorph | stylerMorph classCategory = aCategory].
+	blockStylerMorphsToAdd
+		doWithIndex: [:blockStylerMorph :index | 
+			| positionedMorph |
+			positionedMorph := blockStylerMorph position: 20 @ (index - 1 * blockStylerMorph height).
```

Comment:
Magic Number 20

Reply:
Position for the first object seems to be `blockStylerMorph height` but should be 20 as well maybe?!
Maybe `(index - 1 * blockStylerMorph height + 20` and replace `20` with the constant for the magic number.